hkpy/hkbase/observer/clients/__init__.py

```diff
@@ -0,0 +1,9 @@
+###
+# Copyright (c) 2019-present, IBM Research
+# Licensed under The MIT License [see LICENSE for details]
+###
+
+from .observerclient import ObserverClient as DefaultObserverClient
+from .configurableobserverclient import ConfigurableObserverClient
+from .rabbitmqobserverclient import RabbitMQObserverClient
+from .restobserverclient import RESTObserverClient
```

Comment:
Module names should be snake case, e.g.,  `configurableobserverclient` ==> `configurable_observer_client`